core/core.go

```diff
@@ -99,6 +143,11 @@ func Run() error {
 
 	// Plugins need to start after taskman is running, because taskman provides the FID
 	integration.PluginsInstance().InitAll(state.taskman.GetFrameworkID())
+	runMetrics()
+	defer ecsmetrics.StopGolangMetrics()
+	defer monitoring.Stop()
+
+	log.Infof("Everything initiated Listening on control port: %d", viper.GetInt("controlPort"))
```

Comment:
i don't think operators care about which ports are being listened on. imho, you could consider setting it to info/support.